modules/rosetta/src/test/scala/org/tessellation/rosetta/domain/construction/ConstructionServiceSuite.scala

```diff
@@ -44,4 +59,51 @@ object ConstructionServiceSuite extends MutableIOSuite with Checkers {
       expect.eql(Left(InvalidPublicKey), _)
     }
   }
+
+  test("Successfully deserialize a transaction from hex") { res =>
```

Comment:
I'd change its name to something like:
```
returns transaction hash for a valid transaction hex
```

Test is a specification. You don't want to expose too much details in the specification, especially implementation details. Test should pass as long as I put the transaction hex and receive a hash. I don't care whether it deserialize a transaction or do quantum physics :)

btw: in the whole codebase we write tests from lower cases.

Reply:
Changed.